ai2thor/tests/test_unity.py

```diff
@@ -1279,4 +1278,32 @@ def test_get_coordinate_from_raycast(controller):
     assert_near(
         query.metadata["actionReturn"],
         {'x': -0.5968407392501831, 'y': 1.5759981870651245, 'z': -1.0484200716018677}
-    )
\ No newline at end of file
+    )
+
+
+@pytest.mark.parametrize("controller", fifo_wsgi)
+def test_get_reachable_positions(controller):
```

Comment:
It looks like a test with this method name already exists.  Can you rename the new one?

Reply:
Done